ext/background/const.js

```diff
@@ -5,6 +5,15 @@ const BLOCK_LIST = { title: [/^chrome:\/\//, /^Range/] };
 const GUID_REGEX =
   '({){0,1}[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}(}){0,1}';
 
+const DEDUPE_BEHAVIOR = {
```

Comment:
I don't see this as "deduplication" but merging strategy. It's similar to git where you have `git merge --ours` and `git merge --theirs` since you are describing what happens when there's a conflict.

How about: `MERGE_EXISTING`, `MERGE_NEW`, and `REPLACE`.

Reply:
I agree, that's clearer.